app/controllers/reviews_controller.rb

```diff
@@ -1,4 +1,4 @@
-
+require 'pry'
 
```

Comment:
If you're using the `pry-rails` gem (and your Gemfile confirms that you are) you should not need to `require 'pry'` this way.